docs/source/mro.rst

```diff
@@ -0,0 +1,47 @@
+
+In addition to installing R packages from conda channels, it is possible to install R packages from the Comprehensive R Archive Network (CRAN) or the Microsoft R Application Network (MRAN). These R packages will install into the currently active conda environment. MRO includes the `checkpoint package <https://github.com/RevolutionAnalytics/checkpoint/>`_, which installs from MRAN and is designed to offer enhanced `reproducibility <https://mran.revolutionanalytics.com/documents/rro/reproducibility/>`_.
+
+To use MRO or R packages, activate the conda environment where they are installed to set your environment variables properly. Executing the programs at the pathname in that environment without activating the environment may result in errors.
+
+Each conda environment may have packages installed from the channel "r" or the channel "mro", but no conda environment should contain packages from both channels, which may result in errors. If this occurs, you may use ``conda remove`` to remove the packages that were installed incorrectly, or create a new environment and install the correct packages there.
+
+Installing the Intel Math Kernel Library (MKL) with Microsoft R Open
+====================================================================
+
+The Intel Math Kernel Library (MKL) extensions are also available for MRO on Windows and Linux, while the Accelerate library is used instead on OS X. Just `download the MKL package for your platform <https://mran.revolutionanalytics.com/download/>`_ and install it according to the instructions below.
+
+NOTE: By using our install process, we assume that you have already agreed to the `MKL license <https://mran.revolutionanalytics.com/assets/text/mkl-eula.txt>`_. Proceeding further is an implicit agreement to said license.
+
+When MKL is not installed, each time the R interpreter starts it will display a message saying so, such as::
+
+  No performance acceleration libraries were detected. To take advantage of the available processing power, also install MKL for MRO 3.2.3. Visit http://go.microsoft.com/fwlink/?LinkID=698301 for more details.
+
+When the MKL extensions are properly installed, each time the R interpreter starts it will display a message stating that MKL is enabled, such as::
+
+  Multithreaded BLAS/LAPACK libraries detected.
+
+(BLAS/LAPACK libraries such as MKL are implementations of the Basic Linear Algebra Subprograms specification of the LAPACK Linear Algebra PACKage.)
+
+MKL installation on Linux
+-------------------------
+
+1. Download and extract the proper MKL package to a temporary folder, which we will call $REVOMATH .
+
+   NOTE: You must select the correct MKL package for your Linux distribution.
+
+2. Determine where R is located. The path will have the form of ``/path/to/anaconda/envs/<r environment>`` , which we will call $PREFIX . Ensure that you have write permissions to $PREFIX.
+3. Make backups of ``$PREFIX/R/lib/libRblas.so`` and ``$PREFIX/R/lib/libRlapack.so`` .
```

Comment:
These paths were changed to `$PREFIX/lib/R/lib/libRblas.so` and `$PREFIX/lib/R/lib/libRlapack.so`